cli/build.rs

```diff
@@ -1,17 +1,49 @@
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use std::{env, fs};
+use std::vec::Vec;
 
 fn main() {
     if let Some(git_sha) = read_git_sha() {
         println!("cargo:rustc-env={}={}", "BUILD_SHA", git_sha);
     }
 
+    if wasm_files_present() {
+        println!("cargo:rustc-cfg={}", "TREE_SITTER_EMBED_WASM_BINDING");
+    }
+
     println!(
         "cargo:rustc-env=BUILD_TARGET={}",
         std::env::var("TARGET").unwrap()
     );
 }
 
+#[cfg(unix)]
+fn required_files() -> Vec<&'static Path> {
+    return vec![
+        Path::new("../cli/src/web_ui.html"),
+        Path::new("../docs/assets/js/playground.js"),
+        Path::new("../lib/binding_web/tree-sitter.js"),
+        Path::new("../lib/binding_web/tree-sitter.wasm")
+        ];
+}
```

Comment:
The files `web_ui.html` and `playground.js` are actually always present, and checked into the repo. So I think you only need to check for the existence of `tree-sitter.js` and `tree-sitter.wasm`.

I actually don't think you need any platform-specific logic in this script. On Windows, `wasm_files_present` will return false, because we don't build the wasm binding on Windows (at least not on CI). If some windows user *does* have a working emscripten toolchain, they should be able to build Tree-sitter with the wasm binding embedded.